data-integration/utilities.py

```diff
@@ -0,0 +1,218 @@
+):
+    """
+    Load admin_units to pgsql.
+
+    Parameters
+    ----------
+    rasterdf : GeoDataFrame
+        Table with all rasters to be loaded
+    dbURL : string, optional
+        SQLAlchemy database URL. The default is 'postgresql://postgres:postgres@localhost:5432/dataset'.
+
+    Returns
+    -------
+    None.
+
+    """
+    db_engine = sqla.create_engine(dbURL)
+    logging.info("Loading to PostgreSQL...")
+    data.to_sql(schema, db_engine, if_exists="append", index=False)
+    logging.info("Done.")
+
+
+def toPostGIS(
+    gdf, dbURL="postgresql://postgres:postgres@localhost:5432/dataset", schema="spatial"
+):
+    """
+    Load admin_units to pgsql.
+
+    Parameters
+    ----------
+    admin_units : GeoDataFrame
+        Table with all administrative units..
+    dbURL : string, optional
+        SQLAlchemy database URL. The default is 'postgresql://postgres:postgres@localhost:5432/dataset'.
+
+    Returns
+    -------
+    None.
+
+    """
+    db_engine = sqla.create_engine(dbURL)
+    logging.info("Loading to PostGIS...")
+    gdf.to_postgis(schema, db_engine, if_exists="append", index=False)
+    logging.info("Done.")
+
+
+def datasetExists(
+    ds_id,
+    dbURL="postgresql://test:example@localhost:5433/dataset",
+    tables=["datasets", "spatial", "data"],
+):
+    """
+    Check whether the datasets exist in all tables.
+    Parameters
+    ----------
+    ds_id : int.
+    dbURL : str, optional
+        The default is "postgresql://test:example@localhost:5433/dataset".
+    tables : list of strings, optional
+        The default is ["datasets","spatial","data"].
+    Returns
+    -------
+    bool
+    """
+    engine = sqla.create_engine(dbURL)
+    lengths = []
+    for table in tables:
+        with engine.connect() as con:
+            rs = con.execute(
+                "SELECT COUNT(*) FROM {} WHERE ds_id = {}".format(table, ds_id)
+            )
+            for row in rs:
+                count = row[0]
+            lengths.append(count)
+    if lengths[0] > 0 or lengths[1] > 0 or lengths[2] > 0:
+        return True
+    else:
+        return False
+
+
+def removeDataset(ds_id, dbURL="postgresql://test:example@localhost:5433/dataset"):
+    """
+    Delete the dataset.
+    Parameters
+    ----------
+    ds_id : int.
+    dbURL : str, optional
+        The default is "postgresql://test:example@localhost:5433/dataset".
+    Returns
+    -------
+    None
+    """
+    engine = sqla.create_engine(dbURL)
+    with engine.connect() as con:
+        rs = con.execute("DELETE FROM datasets WHERE ds_id = {};".format(ds_id))
+
+
+def getDataPackage(ds_id, dbURL="postgresql://test:example@localhost:5433/dataset"):
+    """
+    Retrieve the datapackage.
+    Parameters
+    ----------
+    ds_id : int.
+    dbURL : str, optional
+        The default is "postgresql://test:example@localhost:5433/dataset".
+    Returns
+    -------
+    datatpackage: json string.
```

Comment:
Maybe replace by "Optional datapackage" or "datapackage or None" as we can also return None